### ISSUE TITLE
"Find me" does not zoom in enough 

### ISSUE THREAD

**Issue by lordyavin** [theCrag/tca#108]
**What happened?**

Yesterday I remembered to use the app to lookup routes at the crag and I used the "Find me" function on the map to quickly select the sector I were located. But it wasn't quick because the app did not zoom in enough. I had to manually increase the zoom level until I got the circle for the sector I was climbing at. 

**What you expected:**

* [ ] zoom until the map scale matches the GPS accuracy 
* [ ] show GPS accuracy 
* [ ] show busy indicator if GPS resolution is not finished

**Comment by bkucsera** [comment]
The automatic zoom has been modified a bit, please check. 
We do not zoom in so much, because most of the user cases are browsing the map for crags. If you click on the map icon in a crag's map, it will zoom you directly above the selected crag/sector.

**Comment by lordyavin** [comment]
Please check the tasks in the issue description that supposed to be done. 

Oh and please, always update the change info for each app update in the store with meaningful infos. Which version does this apply to?

**Comment by bkucsera** [comment]
You are right, i will add the version info as well in the future. This has been modified a while ago, if you download the actual android version, you will see.
According to the other bullet points, i would say we can leave them with the Enchancement label. At the moment we have several higher priority issues to fix.
